asm/non_matchings/code/sins/sins.s

```diff
@@ -0,0 +1,41 @@
+.data
```

Comment:
.data doesn't belong in non matching files, it's better to keep it in a separate .data.s file in data/ until it's decompiled.